input/fsh/DEF_Rulesets.fsh

```diff
@@ -121,3 +121,16 @@ RuleSet: NCHSObservationCommon
 // * effective[x] 1..1
 * effective[x] only dateTime
 * effective[x] ^short = "Date/Time when added to death record"
+
+RuleSet: addentry(type, id)
+* entry[+].resource = {id}
+* entry[=].fullUrl = "http://www.example.org/fhir/{type}/{id}"
+
+RuleSet: addentryComposition(type, id)
+* entry[+].reference = "http://www.example.org/fhir/{type}/{id}"
+
+RuleSet: addNamedEntryComposition(name, type, id)
+* entry[{name}][+].reference = "{type}/{id}"
+
+RuleSet: addReferenceComposition (field, type, id)
+* {field}.reference = "http://www.example.org/fhir/{type}/{id}"
```

Comment:
This seems inconsistent....
Entry references skip the example.org
field references include example.org...

Can they be made consistent?